presto/presto.go

```diff
@@ -832,7 +832,7 @@ func parseType(name string) []string {
 
 // ConvertValue implements the driver.ValueConverter interface.
 func (c *typeConverter) ConvertValue(v interface{}) (driver.Value, error) {
-	switch c.parsedType[0] {
+	switch strings.ToLower(c.parsedType[0]) {
```

Comment:
Please move lowercasing to `parseType` method

Reply:
Done!